src/libraries/System.Collections.NonGeneric/tests/CaseInsensitiveComparerTests.cs

```diff
@@ -64,7 +64,7 @@ public void Ctor_CultureInfo_Compare(object a, object b, int expected)
             }
         }
 
-        [Fact]
+        [ConditionalFact(typeof(PlatformDetection), nameof(PlatformDetection.IsNotHybridGlobalizationOnOSX))]
```

Comment:
What's the reason for blocking? Is `CaseInsensitiveComparer` not supported on iOS? If it is, shouldn't it be marked as active issue that we want to investigate later? I think that the same for all the other conditional blocking. If we plan to investigate them, it's better to mark them with ActiveIssues. If we plan to drop the support for them, the conditional fact/theory + mention in the docs on changed behavior is fine.

Reply:
This is Ctor_CultureInfo_Compare_TurkishI() it will not be supported, will check/update these test cases in case will be investigates will add ActiveIssue, if no will update the doc.